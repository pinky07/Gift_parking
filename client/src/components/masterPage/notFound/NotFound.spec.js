import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

describe('NotFound', () => {

    let actual;
    it('should render the correct title and message', () => {
        actual = renderer.create(<NotFound/>).toJSON();

        expect(actual).toMatchSnapshot();
    });
});