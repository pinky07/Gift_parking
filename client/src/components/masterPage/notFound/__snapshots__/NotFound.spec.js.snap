// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render the correct title and message 1`] = `
<div>
  <h1>
    Page not found
  </h1>
  <p>
    We are sorry but the page you are looking for does not exist.
  </p>
</div>
`;
